notebooks/reinforcement_learning/solutions/contextual_bandits_with_tf_agents.ipynb

```diff
@@ -0,0 +1,1813 @@
+{
```

Comment:
<p>Running Tensorboard in Jupyter notebook doesn't seem to work well since the above command for me resulted in a bash prompt asking me to enter yes/No as. There is no way to answer in jupyter lab directly. So suggestion is to print the tensorboard command using this statement and run it from the terminal</p><pre class="ql-syntax" spellcheck="false">print ("tensorboard dev upload --logdir " + os.path.dirname(os.path.realpath(ROOT_DIR)) + \
&nbsp; &nbsp; &nbsp; &nbsp;""" --name \"Contextual Bandit\" \
&nbsp; &nbsp; &nbsp; &nbsp;--description \"CB Model for Covertype \"""")


</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/160/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='90'/>

Reply:
<p>Done!</p>